fastreplacestring/config/discover.ml

```diff
@@ -0,0 +1,33 @@
+module C = Configurator.V1
+
+let () =
+  C.main ~name:"configure fastreplacestring" (fun c ->
+
+    let flags =
+      match C.ocaml_config_var c "system" with
+      | Some "mingw64" -> [
+          "-ccopt"; "-lstdc++";
+        ]
+      | Some _
+      | None -> [
+          "-ccopt"; "-Ofast";
+          "-ccopt"; "-lstdc++";
+        ]
+    in
+    C.Flags.write_sexp "dune.flags" flags;
+
+    let cxx_flags =
+      match C.ocaml_config_var c "system" with
+      (* do not link statically on macos *)
+      | Some "mingw64" -> [
+        "-lstdc++";
+        "-fno-exceptions";
```

Comment:
I don't think we need `-fno-exceptions` and `-fno-rtti` in this case - but if we do, I think they would need to be prefixed with `ccopt` as the flags above above. (The latest AppVeyor build is complaining about the `-fno-exceptions` and `-fno-rtti` flags).